theClanless/src/main/java/theClanless/actions/DOMCAction.java

```diff
@@ -0,0 +1,63 @@
+package theClanless.actions;
+
+import com.evacipated.cardcrawl.mod.stslib.actions.common.SelectCardsInHandAction;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.*;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.LoseStrengthPower;
+import com.megacrit.cardcrawl.powers.NightmarePower;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import com.megacrit.cardcrawl.powers.ThornsPower;
+import org.graalvm.compiler.core.common.type.ArithmeticOpTable;
+import theClanless.powers.ManeuverPower;
+
+public class DOMCAction extends AbstractGameAction {
+
+    private AbstractPlayer player;
+    private static final float DURATION = 0.5F;
+
+    public DOMCAction(AbstractPlayer p, int amount) {
+        this.duration = 0.5F;
```

Comment:
Should this be initialized to DURATION?

Reply:
the original code would set it with a method, i just did it directly because I didn't need to set the other parameters